ant/__init__.py

```diff
@@ -20,4 +20,6 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+from __future__ import absolute_import, print_function
```

Comment:
As with the pull request in the extractor, these future-stuff are not neccesary in each and every file right?